### ISSUE TITLE
Propagate exceptions from FastKafka to Tester

### ISSUE THREAD

**Issue by davorrunje** [airtai/fastkafka#121]
When running in serve mode, FastKafka should log all exceptions and just continue processing messages.

When running under Tester, FastKafka should propagate its exceptions to Tester.

```
app = FastKafka()

class Data(BaseModel):
  a: str

@app.consumes()
def on_data(data: Data):
  raise ValueError(42)

@app.produces()
def to_db(x: int) -> data: Data:
  return "whatever"

with pytest.raises(ValueError) as e:
   tester = Tester(app)
    with tester.
        tester.to_data(Data("fortytwo")
    
await tester.awaitable_mocks.on_data.assert_raised_exception

assert e.value ==42
```

**Comment by davorrunje** [comment]
I had a quick look through the code snippets and docs so maybe I missed it, what happens if the message received on the topic doesn't match the pydantic model? Is there an error handler or callback?

https://news.ycombinator.com/item?id=35086594

**Comment by davorrunje** [comment]
https://kai-waehner.medium.com/error-handling-via-dead-letter-queue-in-apache-kafka-903794d7cf78

**Comment by davorrunje** [comment]
Closed by #278
